lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,27 @@
-
 def palindrome_permutation?(string)
-  raise NotImplementedError, "palindrome_permutation? not implemented"
+  letters_hash = {}
+  letters_array = string.split("")
```

Comment:
You don't need to split the string.  You can use`.each_chars` to loop through a string letter-by-letter.